src/left-nav.jsx

```diff
@@ -92,6 +93,12 @@ const LeftNav = React.createClass({
      * The width of the `LeftNav` in pixels. Defaults to using the values from theme.
      */
     width: React.PropTypes.number,
+
+    /**
+     * This number represents the zDepth of the menu (default: 2).
```

Comment:
You can event remove `(default: 2)` as our documentation is picking it up automatically.
Sorry, I should have seen this before :grin:.